aws/nlb/nlb.tf

```diff
@@ -0,0 +1,291 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# VARIABLES / LOCALS / REMOTE STATE
+# ----------------------------------------------------------------------------------------------------------------------
+
+variable "vpc_id" {
+  type        = string
+  description = "VPC ID to associate with ALB"
+}
+variable "subnet_ids" {
+  type        = list(string)
+  description = "A list of subnet IDs to associate with ALB"
+}
+
+variable "internal" {
+  type        = bool
+  default     = false
+  description = "A bool flag to determine whether the ALB should be internal"
+}
+variable "http_redirect_enabled" {
+  type        = bool
+  default     = true
+  description = "A bool flag to enable/disable HTTP listener"
+}
+variable "http_ingress_cidr_blocks" {
+  type        = list(string)
+  default     = ["0.0.0.0/0"]
+  description = "List of CIDR blocks to allow in HTTP security group"
+}
+variable "http_ingress_prefix_list_ids" {
+  type        = list(string)
+  default     = []
+  description = "List of prefix list IDs for allowing access to HTTP ingress security group"
+}
+variable "domain_name" {
+  type        = string
+  default     = ""
+  description = ""
+}
+variable "dns_zone_id" {
+  type        = string
+  default     = ""
+  description = ""
+}
+variable "dns_zone_name" {
+  type        = string
+  default     = ""
+  description = ""
+}
+variable "certificate_arn" {
+  type        = string
+  default     = ""
+  description = "The ARN of the default SSL certificate for HTTPS listener"
+}
+variable "https_ports" {
```

Comment:
All these `https*` variables are not in use and should be removed. We will not have HTTPS with the NLB